earth_enterprise/Upgrade.md

```diff
@@ -1,10 +1,10 @@
-# Upgrading Earth Enterprise Fusion and Server to 5.2.1
+# Upgrading Earth Enterprise Fusion and Server to version 5.2.1
 
-Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.1.3 and 5.2.0 versions.
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
 
-## Upgrading GEE 5.2.0 to 5.2.1
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
```

Comment:
Suggestion: Include commands that user executes.  This does not make it clear whether you're supposed to be running install scripts, or installing packages, and what to do, if you had installed with install scripts, and now you want to install from packages.

Reply:
You mean include:
```cd earth_enterprise/src/installer
sudo ./install_fusion.sh ```

And
``` cd earth_enterprise/src/installer
sudo ./install_server.sh ```

?

If yes, those commands are outlined in the linked document: https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server